src/savegame_json.cpp

```diff
@@ -1913,6 +1913,11 @@ void vehicle::deserialize(JsonIn &jsin)
             }
         }
     }
+
+    for( auto turret : get_parts( "TURRET", false ) ) {
+        install_part( turret->mount.x, turret->mount.y, static_cast<vpart_id>( "turret_mount" ), false );
```

Comment:
This is a unusual way of creating a `vpart_id` instance. Normally one just creates a temporary object directly: `vpart_id( "the-id" )`.